scalastyle-config.xml

```diff
@@ -86,7 +86,7 @@ This file is divided into 3 sections:
   </check>
 
   <check level="error" class="org.scalastyle.scalariform.ObjectNamesChecker" enabled="true">
-    <parameters><parameter name="regex"><![CDATA[[A-Z][A-Za-z]*]]></parameter></parameters>
+    <parameters><parameter name="regex"><![CDATA[(config|[A-Z][A-Za-z]*)]]></parameter></parameters>
```

Comment:
That could be done, but I want this to be the "official" pattern going forward, and having to disable scalastyle every time it's used is kinda ugly and somewhat detracts from the official-ness of it.